uniffi_macros/src/export/callback_interface.rs

```diff
@@ -182,18 +182,21 @@ pub(super) fn metadata_items(
     self_ident: &Ident,
     items: &[ImplItem],
     module_path: &str,
+    docstring: Option<String>,
 ) -> syn::Result<Vec<TokenStream>> {
     let trait_name = ident_to_string(self_ident);
-    let callback_interface_items = create_metadata_items(
-        "callback_interface",
-        &trait_name,
-        quote! {
-            ::uniffi::MetadataBuffer::from_code(::uniffi::metadata::codes::CALLBACK_INTERFACE)
-                .concat_str(#module_path)
-                .concat_str(#trait_name)
-        },
-        None,
-    );
+
+    let mut metadata_expr = quote! {
+        ::uniffi::MetadataBuffer::from_code(::uniffi::metadata::codes::CALLBACK_INTERFACE)
+            .concat_str(#module_path)
+            .concat_str(#trait_name)
+    };
+    if let Some(docstring) = docstring {
+        metadata_expr.extend(quote! { .concat_str(#docstring) })
+    }
```

Comment:
If adding a docstring is optional, don't we have to encode a bool for whether there is a docstring or not?

Reply:
Well, I should really pass a String and not an Option<String>, as this is only called for callback interfaces, and `MetadataReader::read_callback_interface` always read it.

I should unwrap the option in the `ExportItem::Trait` match case of `expand_export`. I will fix that.

Reply:
Fully fixed. No more Options.